core/consensus_test.go

```diff
@@ -121,6 +121,23 @@ func quorum(numNodes uint64) uint64 {
 	}
 }
 
+func commonHasQuorumFn(numNodes uint64) func(blockNumber uint64, messages []*proto.Message, msgType proto.MessageType) bool {
+	quorum := quorum(numNodes)
+
+	return func(blockNumber uint64, messages []*proto.Message, msgType proto.MessageType) bool {
+		switch msgType {
+		case proto.MessageType_PREPREPARE:
+			return len(messages) >= 0
```

Comment:
Shouldn't this condition be like this (even for the testing)?

```suggestion
			return len(messages) == 1
```